mne/time_frequency/tests/test_tfr.py

```diff
@@ -421,6 +421,39 @@ def test_plot():
     plt.close('all')
 
 
+def test_plot_joint():
+    """Test TFR plotting."""
+    import matplotlib.pyplot as plt
+
+    raw = read_raw_fif(raw_fname)
+    times = np.linspace(-0.1, 0.1, 200)
+    n_freqs = 3
+    nave = 1
+    rng = np.random.RandomState(42)
+    data = rng.randn(len(raw.ch_names), n_freqs, len(times))
+    tfr = AverageTFR(raw.info, data, times, np.arange(n_freqs), nave)
+
+    tfr.plot_joint(picks=[0, 1], title='title', colorbar=True)
+    plt.close('all')
+
+    fig = tfr.plot_joint(exclude=[raw.ch_names[0]])
+    fig.canvas.key_press_event('up')
+    fig.canvas.key_press_event(' ')
+    fig.canvas.key_press_event('down')
+    fig.canvas.scroll_event(0.5, 0.5, -0.5)  # scroll down
+    fig.canvas.scroll_event(0.5, 0.5, 0.5)  # scroll up
+
+    # TFR - Topomap joint plot
+    timefreqs = [(tfr.times[2], tfr.freqs[1]),
+                 (tfr.times[1], tfr.freqs[1])]
```

Comment:
True, it's pretty arbitrary. I'll make it the first one and the last one (index 0 and -1)  instead to test the range instead, what do you think?